analyticsdataserver/tests/test_clients.py

```diff
@@ -48,7 +69,17 @@ def test_all_videos_401(self, logger):
     @responses.activate
     @mock.patch('analyticsdataserver.clients.logger')
     def test_all_videos_404(self, logger):
-        responses.add(responses.GET, 'http://example.com/blocks/', status=404, content_type='application/json')
+        responses.add(
```

Comment:
I'm suggesting moving the oauth_token endpoint mock to the separate function as it reusable across the tests